tests/e2e/adu/device/e2e_device_commands.c

```diff
@@ -0,0 +1,1247 @@
+/* Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License. */
+
+#include "e2e_device_process_commands.h"
+
+#include <time.h>
+
+#include "azure_iot_http.h"
+#include "azure_iot_transport_interface.h"
+#include "transport_socket.h"
+#include "azure_iot_json_writer.h"
+#include "azure_iot_jws.h"
+#include "azure_iot_hub_client_properties.h"
+#include "azure/core/az_json.h"
+#include "azure/core/az_span.h"
+
+#define e2etestMESSAGE_BUFFER_SIZE                         ( 10240 )
+
+#define e2etestPROCESS_LOOP_WAIT_TIMEOUT_IN_MSEC           ( 2 * 1000 )
+
+#define e2etestE2E_TEST_SUCCESS                            ( 0 )

```

Comment:
I think that's true. I'd like to keep them separate for now so that anything that is now in `/iot` can be assumed to be otherwise untouched. Later we can look to merge/join these up.